packages/atomic/src/components/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -256,4 +257,8 @@ export class AtomicRefineModal implements InitializableComponent {
       </atomic-modal>
     );
   }
+
+  public componentDidLoad() {
+    this.host.style.display = '';
```

Comment:
What is this needed for ?

Reply:
https://github.com/coveo/ui-kit/pull/1984/files#r871352884